src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-3-0-0.mdx

```diff
@@ -0,0 +1,26 @@
+---
+subject: PHP agent
+releaseDate: '2022-11-03'
+version: 10.3.0.0
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.2.0.0'
+---
+
+## New Relic PHP agent v10.3.0.0
+
+### Important end-of-life information
+
+* Support for 32 bit and FreeBSD binaries were deprecated with release v9.19.0.309 and **are no longer shipped starting with this release**.
+* Support for ZTS binaries was deprecated with release v9.17.0.300 and will be **removed in a release in the near future**.
+* **It is essential any installations using an unsupported PHP version or platform (32 bit, FreeBSD, ZTS, etc) DISABLE auto-update mechanisms for the PHP agent.**  Alternately, pinning the PHP agent to version that is known to have support for the now unsupported feature(s) would achieve the same effect.  Failure to do so could result in a newer release (without support for the system) being installed and lead to the disruption of the collection of APM data.
```

Comment:
```suggestion
* **For installations using an unsupported PHP version or platform (32 bit, FreeBSD, ZTS, etc), you must  disable the auto-update mechanisms for the PHP agent.** As an alternative, you could pin the PHP agent to an agent version that supports the old, unsupported feature(s). Failure to do so could result in a newer release being installed without support for the system. This could disrupt APM data collection.
```